cypress/e2e/ui/footer.cy.js

```diff
@@ -1,86 +1,97 @@
 /// <reference types="Cypress" />
 
 import { footer } from "../../support/page-objects/footer.js"
+import { header } from "../../support/page-objects/header.js"
```

Comment:
recommend set up an index file and export all these fixtures from there, this would allow us to import in the following way;

`import { footer, header } from "../../support/page-objects/`